examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -230,6 +232,27 @@ CHIP_ERROR Commands::RunCommand(int argc, char ** argv, bool interactive,
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
 
+    if (strncmp(argv[1], "dumpstate", 9) == 0)
+    {
+        DumpState();
+        return CHIP_NO_ERROR;
+    }
+    if (strncmp(argv[1], "queue", 5) == 0)
```

Comment:
Again: why is this hardcoded instead of being registered as a command?

But also: is this "must explicitly queue" what we really want?  Or should we queue by default but have an optional arg on things to not queue or something?

Reply:
For using a hardcoded is to reuse the command parsing code.

Now, I just reuse the `--lit-icd-peer` flag setting it to `false` will disable queuing this command.

By default, commands will be queued for registered LIT, and will not be registered for other devices.